DOC-2869 Refactored the port information as a pre-step to adding info… (#2954)
RedisLabs/redislabs-docs: content/embeds/port-availability-embed.md

If ports that Redis assigns to the database are being used by the operating system or other processes, the installation fails.

> Follow the relevant sections to configure the minimum required ports.

Reviewer: Not sure about the word "minimum" here.
```suggestion
Follow the relevant sections to configure required ports.
```

> [Network port configurations]({{< relref "/rs/networking/port-configurations.md" >}}) provides additional details on recommended and optional configurations.

Reviewer: ```suggestion
For recommended and optional port configuration, see [Network port configurations]({{< relref "/rs/networking/port-configurations" >}}).
```
